lib/bcdice/game_system/Satasupe.rb

```diff
@@ -47,7 +49,9 @@ class Satasupe < Base
         ・D66ダイスあり
       INFO_MESSAGE_TEXT
 
-      register_prefix('(\d+R|SR\d+|TAGT|\w+IET|\w+IHT|F\w*T|F\w*T|A\w*T|G\w*A\w*T|A\w*T|R\w*FT|NPCT|KusaiMT|EnterT|BudTT|GetgT|GetzT|GetnT|GetkT|GETSST|PayT|TimeUT|\w+IT|ShaokinRET|MinamiRET|ChinatownRET|CivicCenterRET|DowntownRET|WarshipLandRET|LoveLoveRET|AjitoRET|JigokuSpaRET|JailHouseRET).*')
+      register_prefix('\d+R.*', 'SR\\d+.*', 'TAGT.*', 'GETSST.*', 'NPCT.*', TABLES.keys.map { |k| "#{k}.*" }, ALIASES.keys.map { |k| "#{k}.*" })
```

Comment:
```suggestion
      register_prefix('\d+R', 'SR\\d+', 'TAGT', 'GETSST', 'NPCT', TABLES.keys, ALIASES.keys)
```

`register_prefix` では完全一致ではなく先頭一致を確認するため、末尾の `.*` は不要です。